src/frontend/packages/lib_common/src/theme.ts

```diff
@@ -3,6 +3,10 @@ import { ColorType } from 'grommet/utils';
 import { rgba } from 'polished';
 import { css } from 'styled-components';
 
+// Will generate cunningham theme in style.css
+import '../../../node_modules/@openfun/cunningham-react/dist/style.css';
```

Comment:
I guess that's because of some configuration origin, but because I'm curious why can't you use `import '@openfun/cunningham-react/style.css'` ?

Reply:
( https://github.com/openfun/cunningham/blob/8bf13ae3ada2c5e3ac1f5ff1779f4775e4a49f14/packages/react/package.json#L22-L24 )